### DISCUSSION TITLE
Meeting 2024-10-29

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#802] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-10-29)

**Comment by adamgreig** [comment]
Welcome @berkus to the cortex-a team! :tada:

**Comment by adamgreig** [comment]
From last week:

* Cortex-m-rt release? https://github.com/rust-embedded/cortex-m/pull/533
* cortex-m-pac PR https://github.com/rust-embedded/cortex-m/pull/560#event-14579700710
* Proposed new svd2rust methods write_and, write_with_zero_and, modify_and: https://github.com/rust-embedded/svd2rust/pull/874

**Comment by adamgreig** [comment]
Discovery book sprint happened and a bunch of new material was written for the interrupts chapter.

**Comment by korken89** [comment]
If there is time over I'd like to have clarification on our stance on linker symbols in https://github.com/rust-embedded/cortex-m/issues/564 where I'm available for preparing a PR for what the conclusion is :rocket:
